lib/apipie_bindings/api.rb

```diff
@@ -224,7 +224,7 @@ def http_call(http_method, path, params={}, headers={}, options={})
         rescue => e
           clear_credentials if e.is_a? RestClient::Unauthorized
           log.debug e.message + "\n" +
-            inspect_data(e.respond_to?(:response) ? process_data(e.response) : e)
+            (inspect_data(e.respond_to?(:response) ? process_data(e.response) : e)) rescue ''
```

Comment:
If that means no message about the original message gets logged, :-1:. I would like to see the patch, where the original exception (class, message and backtrace), get logged in. I think the patch can look better.